src/prob/test.jl

```diff
@@ -81,58 +81,64 @@
 
 ""
 function _run_mc_ucopf(file, model_type::Type, solver; kwargs...)
-    return _PMs.run_model(file, model_type, solver, _build_mc_ucopf; multiconductor=true, kwargs...)
+    return _PMs.run_model(file, model_type, solver, _build_mc_ucopf; ref_extensions=[ref_add_arcs_trans!], multiconductor=true, kwargs...)
 end
 
 ""
 function _build_mc_ucopf(pm::_PMs.AbstractPowerModel)
-    variable_generation_indicator(pm)
-
-    variable_storage_indicator(pm)
-    variable_storage_energy(pm)
-    variable_storage_charge(pm)
-    variable_storage_discharge(pm)
-    variable_storage_complementary_indicator(pm)
-
-    for c in conductor_ids(pm)
-        variable_voltage(pm, cnd=c)
-
-        variable_generation_on_off(pm, cnd=c)
+    for (n, network) in _PMs.nws(pm)
+        variable_mc_voltage(pm, nw=n)
+        variable_mc_branch_flow(pm, nw=n)
+        variable_mc_transformer_flow(pm, nw=n)
+        variable_mc_generation(pm, nw=n)
+        # variable_generation_indicator(pm)
+        variable_mc_indicator_generation(pm, nw=n)
 
-        variable_active_storage(pm, cnd=c)
-        variable_reactive_storage(pm, cnd=c)
 
-        variable_branch_flow(pm, cnd=c)
-        variable_dcline_flow(pm, cnd=c)
+        constraint_mc_model_voltage(pm, nw=n)
 
-        constraint_model_voltage(pm, cnd=c)
+        variable_storage_indicator(pm)
+        variable_storage_complementary_indicator(pm)
```

Comment:
what about the energy, charge and discharge variables?

Reply:
It seems below you simply used `variable_mc_storage`, does this lump them together?

Reply:
they are variants depending on whether it is actually multiperiod or not